specs/core/1_custody-game.md

```diff
@@ -670,9 +690,7 @@ def process_chunk_challenge_response(state: BeaconState,
     # Verify chunk index
     assert response.chunk_index == challenge.chunk_index
     # Verify bit challenge data is null
-    assert response.chunk_bits_branch == [] and response.chunk_bits_leaf == Hash()
-    # Verify minimum delay
-    assert get_current_epoch(state) >= challenge.inclusion_epoch + ACTIVATION_EXIT_DELAY
```

Comment:
I do not understand the logic behind the minimum delay, so I deleted it. @JustinDrake @vbuterin do you know why this was here?

Reply:
@vbuterin So this was the line we talked about in the call. I think with your commend the idea behind it was that you should not be able to challenge yourself and respond in the same block to get the rewards?
Why can it not just be 1 slot difference then? As in make sure that you can't respond within the same block?